src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -889,6 +893,8 @@ static PyMemberDef TopicPartition_members[] = {
           " :py:const:`OFFSET_STORED`,"
           " :py:const:`OFFSET_INVALID`\n"
         },
+        {"metadata", T_STRING, offsetof(TopicPartition, metadata), READONLY,
```

Comment:
Metadata is binary, so is T_STRING the correct type for both py2 and py3, or should it be a bytes-like thing?

Reply:
Hi Magnus, thanks for reviewing. But if I use T_BYTE here, it can't return the correct metadata. So I still keep it as T_STRING here.

Reply:
But this data should be handled as binary bytes, not a string, so maybe best to check why T_BYTE didnt work.